packages/taquito/src/estimate/rpc-estimate-provider.ts

```diff
@@ -63,6 +64,19 @@ export class RPCEstimateProvider extends OperationEmitter implements EstimationP
   private readonly ORIGINATION_STORAGE = 257;
   private readonly OP_SIZE_REVEAL = 128;
 
+  private async getSignerOrWallet(): Promise<{
```

Comment:
I think this name might get misunderstood in contrast to what it actually returns. Maybe something a bit more general is fine since we don't necessarily have to expose why or how the pkh and the public key gets returned?

My suggestion would be something like `getKeys` or something in a similar light.

Reply:
Thanks for the suggestion. Indeed, getKeys is more descriptive, I will make the change.